pkg/accounting/tally/tally.go

```diff
@@ -85,6 +85,21 @@ func (t *Tally) Tally(ctx context.Context) error {
 		err = t.SaveBWRaw(ctx, tallyEnd, time.Now().UTC(), bwTotals)
 		if err != nil {
 			errBWA = errs.New("Saving for bandwidth failed : %v", err)
+		} else {
+			//remove expired records
+			now := time.Now()
+			_, err = t.bwAgreementDB.GetExpired(tallyEnd, now)
+			if err != nil {
+				return err
+			}
+			var expiredOrdersHaveBeenSaved bool
+			//todo: write files to disk or whatever we decide to do here
```

Comment:
we should make sure there is a ticket in jira for this future work

Reply:
Created https://storjlabs.atlassian.net/browse/V3-1368